src/mimemail.erl

```diff
@@ -1528,6 +1528,16 @@ parse_example_mails_test_() ->
 				?assertEqual(?IMAGE_MD5, erlang:md5(element(5, Image)))
 			end
 		},
+		{"alternative text/html with calendar attachment.",
+			fun() ->
+				Decoded = Getmail("message-text-html-attachment.eml"),
+				?assertMatch({<<"multipart">>, <<"mixed">>, _, _, [{
```

Comment:
minor: I think it will be a bit easier to read if the last symbol on this line `{` is on the line 1535 `{<<"multipart'>, ...` instead

Reply:
fixed in latest push